airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py

```diff
@@ -191,3 +191,10 @@ def extract_image_pull_secrets(pod, req):
     def extract_tolerations(pod, req):
         if pod.tolerations:
             req['spec']['tolerations'] = pod.tolerations
+
+    @staticmethod
+    def extract_env_from(pod, req):
+        for idx, configmap in enumerate(pod.envs_from_configmaps):
+            req['spec']['containers'][0]['envFrom'][idx]['configMapRef']['name'] = configmap.name
```

Comment:
I am not very familiar with Airflow k8s code. But this seems to a long chain of map? Does it always work? Could it happen when the key doesn't exist in the dict which causes exception?

Reply:
@feng-tao oh, You are right. I will fix that.

Reply:
I also don't think your tests are hitting this code path, as you have

```
        configmap = ConfigMapObject(name='test-configmap')
        configmap = """
apiVersion: v1
```